Yoast/wordpress-seo
# Change link on first config wizard screen

## Summary

This PR can be summarized in the following changelog entry:

* None needed.

## Test instructions

This PR can be tested by following these steps:

* Open the config wizard, see the first screen no longer promotes the configuration service but the plugin training. Make sure the plugin training link works and the right image shows, like this:

![yoast_seo_ _configuration_wizard](https://user-images.githubusercontent.com/487629/42132383-19eaf6c2-7d17-11e8-9e49-dfefe534212b.jpg)

## Quality assurance


Fixes #10204
